test/unit/helpers/xhr/POLLING.js

```diff
@@ -0,0 +1,39 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken":"00_J1qxqyLs-6ZutUUWfbqm-1nqnW6n2o5z2wnBRHs",
+    "type":"SESSION_STEP_UP",
+    "expiresAt":"2020-03-23T17:09:34.000Z",
+    "status":"POLL",
+    "_embedded":{
+       "factor":{
+          "id":"okta-poll",
+          "factorType":"okta-poll",
+          "provider":"OKTA",
+          "profile":{
+             "refresh": 1000
```

Comment:
Worth having a refresh value as 2000 or something so we can test the other flow.